FF1Blazorizer/Pages/Randomize.cshtml

```diff
@@ -474,13 +474,21 @@
 		if (seed == _seed)
 			return;
 
+		if (seed.Length > 8)
+		{
+			_seedInputClass = "is-error";
+			return;
+		}
+
 		_seed = seed;
 		try
 		{
 			Blob.FromHex(_seed);
-
-			_seedInputClass = _seed.Length == 8 ? "" : "is-error";
-
+			if (seed.Length < 8)

```

Comment:
I think strings have a PadLeft method you can use to just pad with whatever to a given size. Look elsewhere because I think I do it someplace else in the file already.

Reply:
Done.